{{ cookiecutter.repo_name }}/{{ cookiecutter.repo_name }}/structure.py

```diff
@@ -10,14 +10,13 @@
 from drtools.core.filesystem import FileStructure
 
 PATH = FileStructure(
-        settings.ROOT,
-        {
-            'EVAL':         ('models', ),
-            'DATA':         ('data', ),
-            'RAW':              ('DATA', 'raw', ),
-            'CS_IN':                ('RAW', 'clickstream.csv'),
-            'PROCESSED':        ('DATA', 'processed', ),
-            'CS_OUT':               ('PROCESSED', 'clickstream.pickle'),
-
-        },
-    )
\ No newline at end of file
+    settings.ROOT,
+    {
+        'EVAL':         ('models', ),
+        'DATA':         ('data', ),
+        'RAW':              ('DATA', 'raw', ),
+        'CS_IN':                ('RAW', 'clickstream.csv'),
+        'PROCESSED':        ('DATA', 'processed', ),
+        'CS_OUT':               ('PROCESSED', 'clickstream.pickle'),
+    },
+)
```

Comment:
And what about spaces? Doesn't pycharm underline this for you? Doesn't it disturb you? I can't stand it. :P

Reply:
As you've seen I don't care about what pycharm says here :P I prefer this formatting because it suggests the structure of the directories.

Reply:
see this example it's styled correctly but hard to figure out where each file will end up https://github.com/datarevenue-berlin/rtl_reco/blob/master/rtl_reco/structure.py

Reply:
Wow, I didn't even notice that these indents actually mean something. I thought it's just messed up by tabs or something. Okay, then I can turn a blind eye to it. ;)